client/src/background.js

```diff
@@ -32,10 +34,27 @@ function createWindow() {
     createProtocol('app');
     // Load the index.html when not in development
     win.loadURL('app://./index.html');
+
+    //check for updates
+    autoUpdater.checkForUpdatesAndNotify();
   }
 
+  const ses = win.webContents.session;
+
   win.on('closed', () => {
     win = null;
+    // clean storages after closing app
+    ses.clearStorageData({
+      storages: ['localstorage', 'caches', 'indexdb']
```

Comment:
Lets keep localStorage around so that the user does not have to login again.

Reply:
Changes: The code was documented.